modules/abstract-utxo/src/transaction.ts

```diff
@@ -120,6 +130,12 @@ function explainCommon<TNumber extends number | bigint>(
     outputs.push({
       address: currentAddress,
       amount: currentAmount.toString(),
+      // If changeInfo has a length greater than zero, it means that the change information
```

Comment:
in JS empty objects are truthy, if you want to identify an empty object you can use `Object.keys(changeInfo).length`

Reply:
I need to update this comment - only when changeInfo is undefined do we want to skip this. 

In the case that changeInfo is an empty array - it means that the outputs have been checked internally that they are internal/external but none of them are. This is also the way that we can say that outs are external, not just internal. 

I will update the Jsdoc and this comment to say this, because it isn't obvious